packages/account-tree-controller/src/AccountTreeController.test.ts

```diff
@@ -1558,13 +1567,17 @@ describe('AccountTreeController', () => {
         id: 'unknown-account-id',
       };
 
-      messenger.unregisterActionHandler(
+      accountsControllerMessenger.unregisterActionHandler(
         'AccountsController:getSelectedAccount',
       );
-      messenger.registerActionHandler(
+      accountsControllerMessenger.registerActionHandler(
         'AccountsController:getSelectedAccount',
         () => unknownAccount,
       );
+      messenger.delegate({
+        messenger: accountTreeControllerMessenger,
+        actions: ['AccountsController:getSelectedAccount'],
+      });
```

Comment:
If an action handler is unregistered and then re-registered, the root messenger must delegate the external action to the AccountTreeController messenger again, or the following error will occur: `A handler for AccountsController:getSelectedAccount has not been registered`